--- /home/xor/vextest/venv/lib/python3.12/site-packages/angr/analyses/calling_convention/utils.py
+++ /home/xor/vextest/backups/venv_angr_stack_2026-03-20/patched_files/python3.12/angr/analyses/calling_convention/utils.py
@@ -56,8 +56,8 @@
     if arch_name == "X86":
         return 8 <= reg_offset < 24 or 160 <= reg_offset < 288  # eax, ebx, ecx, edx  # xmm0-xmm7
 
-    if arch_name == "RISCV64":
-        return 96 <= reg_offset < 160  # a0-a7
+    if arch_name == "86_16":
+        return True
 
     l.critical("Unsupported architecture %s.", arch.name)
     return True
